source/views/building-hours/stateful-list.js

```diff
@@ -27,7 +29,31 @@ const groupBuildings = (buildings: BuildingType[]) => {
   return toPairs(grouped).map(([key, value]) => ({title: key, data: value}))
 }
 
-type Props = TopLevelViewPropsType
+const loadAllBuildings = (
+  buildings: BuildingType[],
+  favoriteBuildings: BuildingType[],
+) => {
+  let allBuildings = Array()
+  allBuildings.push({
+    title: 'Favorites',
+    data: favoriteBuildings,
```

Comment:
I'd like to see logic incorporated that restricted the 'Favorites' section from appearing if there are no favorites, if you don't mind. If you don't want to implement this, I'd be happy to work on it.

Reply:
https://github.com/StoDevX/AAO-React-Native/pull/2003/commits/3cdeb6c37bed1768cc0ad1a0e5f0ed6d74cf52d8 
I did it the easy way in this commit, but if you think there's a better way you should go for it. 👍